WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/download/BackupDownloadStates.kt

```diff
@@ -51,27 +59,23 @@ enum class StateType(val id: Int) {
 
 sealed class ToolbarState {
     abstract val title: Int
-    abstract val icon: Int
+    @DrawableRes val icon: Int = R.drawable.ic_arrow_back
```

Comment:
❓ If I recall correctly we wanted to use "close" everywhere, not "back". Or have we decided to switch to "back" icon?

Reply:
🤦 Yes, I left the wrong one in...fantastic catch.